kernel-boot/rdma_rename.c

```diff
@@ -483,6 +483,19 @@ static int by_guid(struct data *d)
 	return 0;
 }
 
+static int set_fixed_name(struct data *d, char *name)
+{
+	int ret = -1;
```

Comment:
No need in assignment, you are overwriting it.

Reply:
thanks, I've updated it